avi20.vcxproj

```diff
@@ -141,6 +142,7 @@
       <PreprocessorDefinitions>AVI20_EXPORTS;_MBCS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ProgramDataBaseFileName>$(OutDir)$(ProjectName).pdb</ProgramDataBaseFileName>
       <LanguageStandard>stdcpplatest</LanguageStandard>
+      <WholeProgramOptimization>false</WholeProgramOptimization>
     </ClCompile>
     <Link>
       <GenerateDebugInformation>true</GenerateDebugInformation>
```

Comment:
can you change to be DebugFull here.

Reply:
![image](https://user-images.githubusercontent.com/8958899/71919572-fecabc00-3152-11ea-84a5-0684745c5ea6.png)